### ISSUE TITLE
Using skipunannotated throws error

### ISSUE THREAD

**Issue by tobiasdiez** [jakebailey/pyright-action#54]
Activating skipunannotated yields
```
Run jakebailey/pyright-action@v1
/usr/bin/docker exec  0570a425a99594c7f7200fb917406d6e67ce96052308507cba4db691694b18bd sh -c "cat /etc/*release | grep ^ID"
pyright 1.1.232, node v16.20.2, pyright-action 1.7.0
/__e/node16/bin/node /__t/pyright/1.1.232/x64/package/index.js --skipunannotated true --outputjson
Loading configuration file at /sage/pyrightconfig.json
Assuming Python platform Linux
stubPath /sage/typings is not a valid directory.
Searching for source files
File or directory "/sage/true" does not exist.
No source files found.
```
It might be because of the old version (that we have to use for some unrelated reason), but I think the args should be simply `--skipunannotated --outputjson` instead of `--skipunannotated true --outputjson`

**Comment by jakebailey** [comment]
This seems to just be a copy paste bug on my part, will fix.

**Comment by jakebailey** [comment]
I won't get to it until the morning, but for now you can work around this with `extra_args`.

**Comment by jakebailey** [comment]
Fixed in 1.7.1.
